tools/cli/go-whisk-cli/commands/action.go

```diff
@@ -382,7 +382,8 @@ func parseAction(cmd *cobra.Command, args []string) (*whisk.Action, error) {
   action := new(whisk.Action)
   action.Name = qName.entityName
   action.Namespace = qName.namespace
-  action.Limits = getLimits()
+  action.Limits = getLimits(cmd.LocalFlags().Changed("memory"), cmd.LocalFlags().Changed("logsize"),
```

Comment:
Since these flag names are also used to define the command flags, these flag labels should be defined as constants and used in both places.